src/main/java/teammates/ui/controller/StudentCourseJoinAuthenticatedAction.java

```diff
@@ -24,8 +25,16 @@
     @Override
     protected ActionResult execute() throws EntityDoesNotExistException {
         Assumption.assertNotNull(regkey);
```

Comment:
If this parameter can contain more than the key, then may be we should not call it regkey?